security/crowdsec/src/opnsense/scripts/OPNsense/CrowdSec/hub-upgrade.sh

```diff
@@ -1,17 +1,16 @@
 #!/bin/sh
 
+/usr/local/bin/cscli --error hub update \
+    && /usr/local/bin/cscli --error hub upgrade
+
 if [ ! -e "/usr/local/etc/crowdsec/collections/opnsense.yaml" ]; then
     /usr/local/bin/cscli --error collections install crowdsecurity/opnsense
 fi
 
-/usr/local/bin/cscli --error hub update \
-    && /usr/local/bin/cscli --error hub upgrade
-
 if service crowdsec enabled; then
-    # have to check status explicitly because "restart" can set $? = 0 even when failing
     if ! service crowdsec status >/dev/null 2>&1; then
         service crowdsec start >/dev/null 2>&1 || :
     else
-        service crowdsec restart >/dev/null 2>&1 || :
+        service crowdsec reload >/dev/null 2>&1 || :
```

Comment:
make sure that reload works across binary updates, e.g. for syslog-ng that is not the case doing funky stuff with shared library object loading on demand

Reply:
thanks for noticing. In case of binary update, this is not the only script that gets called so it should be ok, but I'll test

Reply:
in case of binary upgrade, the daemon is restarted (not reloaded) in the pkg-install, pkg-deinstall scripts of its freebsd package